docs/grid_model/network_subnetwork.md

```diff
@@ -807,6 +807,7 @@ A busbar section is a non impedant element used in a node/breaker substation top
 ## Breaker/switch
 
 [![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Switch.html)<br>
+A switch to connect equipments in a substation.
```

Comment:
"equipment" is an uncountable noun. Could you find another sentence here?

Reply:
It's corrected, thanks.